dataactbroker/handlers/accountHandler.py

```diff
@@ -660,52 +660,54 @@ def listUsersWithStatus(self):
-            submissionDetails.append({"submission_id": submission.submission_id, "last_modified": submission.updated_at.strftime('%m/%d/%Y'),
-                                      "size": total_size, "status": status, "errors": error_count, "reporting_start_date": str(submission.reporting_start_date),
-                                      "reporting_end_date": str(submission.reporting_end_date), "user": {"user_id": submission.user_id,
-                                                                                                    "name": submission_user_name}})
-        return JsonResponse.create(StatusCode.OK, {"submissions": submissionDetails})
-
-    def listSubmissionsByCurrentUser(self):
-        """ List all submission IDs associated with the current user ID """
-        user_id = LoginSession.getName(flaskSession)
-        user = self.interfaces.userDb.getUserByUID(user_id)
-        submissions = self.interfaces.jobDb.getSubmissionsByUserId(user_id)
-        submissionDetails = []
-        for submission in submissions:
-            job_ids = self.interfaces.jobDb.getJobsBySubmission(submission.submission_id)
-            total_size = 0
-            for job_id in job_ids:
-                file_size = self.interfaces.jobDb.getFileSize(job_id)
-                total_size += file_size if file_size is not None else 0
-
-            status = self.interfaces.jobDb.getSubmissionStatus(submission.submission_id)
-            error_count = sumNumberOfErrorsForJobList(submission.submission_id)
-            submissionDetails.append(
-                {"submission_id": submission.submission_id, "last_modified": submission.updated_at.strftime('%m/%d/%Y'),
-                 "size": total_size, "status": status, "errors": error_count, "reporting_start_date": str(submission.reporting_start_date),
-                                      "reporting_end_date": str(submission.reporting_end_date), "user": {"user_id": str(user_id),
-                                                                                                    "name": user.name}})
-        return JsonResponse.create(StatusCode.OK, {"submissions": submissionDetails})
+            submission_details.append({"submission_id": submission.submission_id,
+                                       "last_modified": submission.updated_at.strftime('%m/%d/%Y'),
+                                       "size": total_size, "status": status, "errors": submission.number_of_errors,
+                                       "reporting_start_date": str(submission.reporting_start_date),
+                                       "reporting_end_date": str(submission.reporting_end_date),
+                                       "user": {"user_id": submission.user_id,
+                                                "name": submission_user_name}})
+
+        total_submissions = GlobalDB.db().session.query(Submission).filter(Submission.user_id == user_id).count()
```

Comment:
Not at all important, but in case it makes querying easier in the future, you can use `filter_by`:
```python
total_submissions = GlobalDB.db().session.query(Submission).filter_by(user_id=user_id).count()
```

Love seeing the simplicity here, though.